Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -65,22 +65,29 @@ the token returned by ``kubeadm init``
     name on each node.
 
 For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
-by using the following commands below. **Careful:** Be aware that this will break
-existing service connections. It will also stop service related traffic until the
-Cilium replacement has been installed:
+by using the following commands below.
 
-.. code-block:: shell-session
+.. warning::
+   Be aware that this will break existing service connections. It will also stop service related traffic
+   until the Cilium replacement has been installed:
+
+   .. code-block:: shell-session
 
-    $ kubectl -n kube-system delete ds kube-proxy
-    $ # Delete the configmap as well to avoid kube-proxy being reinstalled during a Kubeadm upgrade (works only for K8s 1.19 and newer)
-    $ kubectl -n kube-system delete cm kube-proxy
-    $ # Run on each node with root permissions:
-    $ iptables-save | grep -v KUBE | iptables-restore
+       $ kubectl -n kube-system delete ds kube-proxy
+       $ # Delete the configmap as well to avoid kube-proxy being reinstalled during a Kubeadm upgrade (works only for K8s 1.19 and newer)
+       $ kubectl -n kube-system delete cm kube-proxy
+       $ # Run on each node with root permissions:
+       $ iptables-save | grep -v KUBE | iptables-restore
```

Comment:
The commands are the logical continuation of the paragraph, so they should be outside of the warning.